src/types/member.ts

```diff
@@ -0,0 +1,11 @@
+import { IconName } from '@components';
+
+export interface IMember {
```

Comment:
Can we deduct the type from the config instead?

Reply:
Per conversation with @vilsbole we decided to rename interface to `ITeamMember`